tests/BinaryOperationTest.php

```diff
@@ -85,6 +85,75 @@ public function testGMPOperations(): void
         $this->assertSame($assertions, $actual_vars);
     }
 
+    public function testDecimalOperations(): void
+    {
+        if (!class_exists('Decimal\\Decimal')) {
+            $this->markTestSkipped('Cannot run test, base class "Decimal\\Decimal" does not exist!');
+        }
```

Comment:
Would you mind adding this extension to the GHA pipeline? It should be rather easy, see https://github.com/shivammathur/setup-php#heavy_plus_sign-php-extension-support

Reply:
I think it's added, but it doesn't seem to want to run since there are changes requested, so I'm going to go ahead and resolve this to see if it works.

Reply:
The logs for Unit Tests - 5 includes:
```
  PHPUnit 9.5.10 by Sebastian Bergmann and contributors.
  
  Runtime:       PHP 8.0.13
  Configuration: /home/runner/work/psalm/psalm/phpunit.xml.dist
  Random Seed:   1638987602
  
  ................................................................. 65 / 77 ( 84%)
  ............                                                      77 / 77 (100%)
  
  Time: 00:33.093, Memory: 60.00 MB
  
  OK (77 tests, 100 assertions)
```
So I think this is fixed.

Reply:
Not sure why the Windows one is failing, the logs show that the extension was installed fine. Is it ok to just disable it on the Windows test? Really I'd like to get around to #5482 sometime and make the test always run whether or not the extension is installed, it doesn't actually need the extension anyway, just the stub. Unfortunately right now, the stub is only loaded if the extension is installed.

Reply:
> Is it ok to just disable it on the Windows test? 

Sure, feel free to do that. I just wanted to have that test run in at least one environment, it doesn't have to be all.